convert.c

```diff
@@ -969,6 +1092,32 @@ static int ident_to_worktree(const char *path, const char *src, size_t len,
 	return 1;
 }
 
+static struct encoding *git_path_check_encoding(struct attr_check_item *check)
+{
+	const char *value = check->value;
+	struct encoding *enc;
+
+	if (ATTR_UNSET(value))
+		return NULL;
+
+	for (enc = encoding; enc; enc = enc->next)
+		if (!strcmp(value, enc->name))
+			return enc;
+
+	/* Don't re-encode to the default */
```

Comment:
According to the [Chicago Manual of Style](http://www.chicagomanualofstyle.org/16/images/ch07_tab01.pdf#page9) and the Merriam-Webster dictionary, prefixes like `multi`, `anti`, `re`, `non`, etc. shouldn’t be separated with a hyphen.